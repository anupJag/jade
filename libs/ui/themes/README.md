# fe-themes

This library was generated with [Nx](https://nx.dev).

## Running unit tests

Run `ng test fe-themes` to execute the unit tests via [Jest](https://jestjs.io).
