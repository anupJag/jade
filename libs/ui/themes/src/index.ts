export * from './lib/defaultTheme';
export * from './lib/styled';
export * from './lib/themeProvider';
export { default as styled } from './lib/styled';
export * from './lib/GlobalStyles';
