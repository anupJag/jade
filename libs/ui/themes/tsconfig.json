{
  "extends": "../../../tsconfig.json",
  "compilerOptions": {
    "jsx": "react",
    "allowJs": false,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "types": ["node", "jest"]
  },
  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"]
}
