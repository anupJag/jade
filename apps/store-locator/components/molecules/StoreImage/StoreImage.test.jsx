import React from 'react';
import { render, screen, waitFor } from '@testing-library/react';
import { ThemeProvider } from '@jade/ui/themes';

import { StoreImage } from './index';

describe('StoreImage', () => {
  let StoreImageComponent = '';

  beforeEach(() => {
    StoreImageComponent = render(
      <ThemeProvider>
        <StoreImage />
      </ThemeProvider>,
    );
  });

  it('should render store basic details correctly', async () => {
    const { asFragment } = StoreImageComponent;
    expect(asFragment()).toMatchSnapshot();
  });
});
