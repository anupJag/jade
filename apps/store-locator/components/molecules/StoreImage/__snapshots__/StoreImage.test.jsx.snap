// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoreImage should render store basic details correctly 1`] = `
<DocumentFragment>
  <div
    class="css-0"
  >
    <img
      alt="image alt"
      class="css-457lt1"
    />
  </div>
</DocumentFragment>
`;
